install_sct

```diff
@@ -335,25 +351,36 @@ for arg in "$@"; do
   esac
 done
 
-while getopts ":dhbpv" opt; do
+SCT_INSTALL_TYPE=""
+NONINTERACTIVE=""
+NO_DATA_INSTALL=""
+NO_SCT_BIN_INSTALL=""
```

Comment:
We set a bunch of other variables higher up. Would it be worth initializing these there, too?

https://github.com/neuropoly/spinalcordtoolbox/blob/281e85da979e5973653584da7370d74d39ab7470/install_sct#L40-L48

Reply:
Hmmm you're probably right.

Reply:
Fixed in a618ee30b3484fb0da2cbc95df701f858140d4ad